README.md

```diff
@@ -55,7 +55,7 @@ trunk serve
 	* [ ] [Working with events](https://docs.mapbox.com/mapbox-gl-js/api/map/#instance-members-working-with-events)
 	* [ ] [Sources](https://docs.mapbox.com/mapbox-gl-js/api/map/#instance-members-sources)
 	* [x] [Images](https://docs.mapbox.com/mapbox-gl-js/api/map/#instance-members-images)
-	* [ ] [Layers](https://docs.mapbox.com/mapbox-gl-js/api/map/#instance-members-layers)
+	* [/] [Layers](https://docs.mapbox.com/mapbox-gl-js/api/map/#instance-members-layers)
```

Comment:
is this a typo?

Reply:
It misses a bunch of methods. I felt a bit uncomfortably implementing them since they're not used in any examples and there wouldn't be any testing on them either. I figured crossing it off entirely would not be right and left this as a placeholder to myself to decide what to do :sweat_smile: I forgot about it instead. wdyt is the correct thing? Maybe just cross it off?

Reply:
It already has a good support of layers. Of course it is not perfect, but we can incrementally add features.  

For me it is okay!